x/feeabs/ibc_module.go

```diff
@@ -201,7 +201,18 @@ func (am IBCModule) OnTimeoutPacket(
 	packet channeltypes.Packet,
 	relayer sdk.AccAddress,
 ) error {
-	// TODO: Resend request if timeout
-	// TODO: emit event
+	chancap := am.keeper.GetCapability(ctx, host.ChannelCapabilityPath(am.keeper.GetPort(ctx), am.keeper.GetChannelId(ctx)))
```

Comment:
We not use this GetPort(ctx) and GetChannelId(ctx). ICQ port ID and Channel ID should in the params. You can reference to these [code](https://github.com/notional-labs/fee-abstraction/blob/3c6994e806a0e4abd50eccff46418c45615a39f9/x/feeabs/keeper/ibc.go#L98-L101) should create a issue to remove these func